jax/_src/numpy/fft.py

```diff
@@ -17,16 +17,20 @@
 
 from jax import lax
 from jax.lib import xla_client
+from jax.api_util import _ensure_index_tuple
+from jax._src.util import safe_zip
 from .util import _wraps
 from . import lax_numpy as jnp
 from jax import ops as jaxops
 
 
 def _fft_core(func_name, fft_type, a, s, axes, norm):
-  # TODO(skye): implement padding/cropping based on 's'.
   full_name = "jax.numpy.fft." + func_name
+
   if s is not None:
-    raise NotImplementedError("%s only supports s=None, got %s" % (full_name, s))
+    s = _ensure_index_tuple(list(s))
```

Comment:
SInce we're disallowing the `isinstance(s, int)` case, I think `s = tuple(map(operator.index, s))` or `tuple(operator.index(n) for n in s)` would be clearer.